lib/saml/url_service.rb

```diff
@@ -32,6 +32,9 @@ def initialize(saml_settings, session: nil, user: nil, params: {})
 
       @saml_settings = saml_settings
 
+      Raven.extra_context(saml_settings: @saml_settings)
```

Comment:
this line might be dangerous to record in Sentry if we're not scrubbing for it in PIISanitizer